public/app/percona/add-instance/components/AzureDiscovery/components/Instances/Instances.types.ts

```diff
@@ -0,0 +1,8 @@
+import { Instance } from '../../Discovery.types';
+
+export interface InstancesTableProps {
+  instances: Instance[];
+  selectInstance: (any) => void;
+  loading: boolean;
+  credentials: any;
```

Comment:
Use the type created for credentials here

Reply:
Left it for future refactoring,

Reply:
Why would we leave it for the future? It'll stay that way forever :)